charts/fluent-operator/templates/fluentd-fluentd.yaml

```diff
@@ -13,6 +13,9 @@ spec:
         port: {{ .Values.fluentd.port }}
   replicas: {{ .Values.fluentd.replicas }}
   image: {{ .Values.fluentd.image.repository }}:{{ .Values.fluentd.image.tag }}
+  {{- if .Values.fluentd.imagePullSecrets }}
+  imagePullSecrets:
+  {{ toYaml .Values.fluentd.imagePullSecrets | indent 4 }}
```

Comment:
```
  {{- if .Values.fluentd.imagePullSecrets }}
  imagePullSecrets:
  {{ toYaml .Values.fluentd.imagePullSecrets | indent 4 }}
{{end}}
```
There seems to be an `end` missing here.

Reply:
@wenchajun Added, Thanks